src/libomemo.c

```diff
@@ -1,3 +1,18 @@
+/**
+ * LIBOMEMO 0.5.0
```

Comment:
Well, it's just one file. And I thought it's nice to have some sort of version indication aside from git, in case git dotfiles get lost when copying or something.